### PR TITLE
[22.x] Bump version to 22.1rc2 & add release notes

### PR DESCRIPTION
**Pull request by fanquake**
Bump the version to 22.1rc2.
Regenerate the man pages.
Add WIP 22.1 release notes.

Changes since rc1:
- https://github.com/bitcoin-core/gui/pull/631
- https://github.com/bitcoin-core/gui/pull/680

### CODE REVIEW

**Comment by hebasto** [comment]
There are no translation updates since #26431.

File: doc/release-notes.md [hunk 1]
```diff
@@ -66,57 +51,75 @@ Updated settings
...
+Kittywhiskers Van Gogh
+Marco Falke
+Martin Zumsande
+Michael Ford
+muxator
+Pieter Wuille
+Ryan Ofsky
+Saibato
+Sebastian Falbesoner
+W. J. van der Laan
```
Comment (hebasto) [inline doc/release-notes.md:122]: This is rendered as one line. Should it be multi-line?

**Review by hebasto — APPROVED** [review]
ACK 72c3eaf300452ae166f20af7c84c2d02ae392aaf

**Review by hebasto — APPROVED** [review]
re-ACK 4bbab04fdca2c11a7d52cd5b2d1b110c43fc2674

File: doc/release-notes.md [hunk 2]
```diff
@@ -42,21 +42,6 @@ From Bitcoin Core 22.0 onwards, macOS versions earlier than 10.14 are no longer
 Notable changes
```
Comment (stickies-v) [inline doc/release-notes.md:42]: https://github.com/bitcoin/bitcoin/pull/25242 seems to be missing - not sure if on purpose?

**Review by stickies-v — COMMENTED** [review]
I verified that as outlined in https://github.com/bitcoin/bitcoin/blob/master/doc/release-process.md:
- the manpages were updated correctly
- the rc version in configure.ac was updated correctly
- the changes since `v22.0` match with what is listed in `release-notes.md`, and found 1 missing PR that I outlined in a comment to this review

I did not verify the translations.

File: doc/release-notes.md (hunk 2 above)
Comment (fanquake) [reply doc/release-notes.md:42]: It's on line 58.

Comment (stickies-v) [reply doc/release-notes.md:42]: Sorry I copied the wrong PR number, I meant #24256

Comment (fanquake) [reply doc/release-notes.md:42]: Included that now.

**Review by stickies-v — APPROVED** [review]
ACK [d9bd628](https://github.com/bitcoin/bitcoin/commit/d9bd628ac9d1e6272cb2f8f67b86376a13233f90)

**Review by jarolrod — COMMENTED** [review]
ACK d9bd628ac9d1e6272cb2f8f67b86376a13233f90

**Comment by MarcoFalke** [comment]
Thanks, Concept ACK

**Pull request merged**